main.js

```diff
@@ -530,45 +643,77 @@ const renderReviewForm = () => {
   `;
 
   printToDom("reviewFormDiv", reviewsFormTemplate);
-  document.getElementById("submit").addEventListener('click', getReviewInfo);
-  document.getElementById("reviewForm").addEventListener('click', addClicksToRadios);
-
-
-}
-
+  document.getElementById("submit").addEventListener("click", getReviewInfo);
+  document
+    .getElementById("reviewForm")
+    .addEventListener("click", addClicksToRadios);
+};
 
 const addClicksToRadios = (e) => {
   let classes = e.target.className;
   console.log(classes);
   let radioClass = classes.replace("avatarStyle avatar-", "");
+<<<<<<< HEAD
+  var ele = document.getElementsByTagName("input");
+  document.getElementById(radioClass).checked = true;
+  if (radioClass != 1) {
+    document.getElementById(1).checked = false;
+  }
+};
+=======
   //var ele = document.getElementsByTagName('input');
   document.getElementById(radioClass).checked = true;
   console.log(document.getElementById(radioClass));
 }
 
+>>>>>>> master
 
 const getReviewInfo = (e) => {
   e.preventDefault();
   let buttonId = e.target.id;
   if (buttonId === "submit") {
+<<<<<<< HEAD
+    var ele = document.getElementsByTagName("input");
+
+    //If select comes out as 3 it is 4, if 0 it is 1
+    let rating = document.getElementById("numberOfStars").selectedIndex + 1;
+    let avatar;
+    // LOOP THROUGH EACH ELEMENT.
+
+=======
     var ele = document.getElementsByTagName('input');
     let rating = document.getElementById('numberOfStars').selectedIndex + 1;
     let avatar;
+>>>>>>> master
     let product = document.getElementById("product").value;
     let name = document.getElementById("name").value;
+<<<<<<< HEAD
+
+    for (let i = 0; i < ele.length; i++) {
+      if (ele[i].type == "radio" && ele[i].checked) {
+        console.log(ele[i].checked);
+        avatar = `${ele[i].id}.jpg`;
+      }
+      // CHECK THE ELEMENT TYPE.
+    }
+    let comment = document.getElementById("comment").value;
+    console.log("All the values current:");
+    console.log(name, product, rating, avatar, comment);
+=======
     for (let i = 0; i < ele.length; i++) {
       if (ele[i].type == 'radio' && ele[i].checked) {
         console.log(ele[i].checked);
         avatar = `${ele[i].id}.jpg`;
       }
     }
     let comment = document.getElementById("comment").value;
+>>>>>>> master
```

Comment:
This shows a merge conflict.

Reply:
Seeing lots of these lines means merge conflicts. Gitkraken is the easier way to solve these. Let me know if I can be of any help with that!